src/utils/search/get-matches.js

```diff
@@ -0,0 +1,21 @@
+import buildQuery from "./build-query";
```

Comment:
LGTM. I was planning to do some opts as next step. 
1. Use transfer objects to transfer matches from worker to main
2. Make next and previous to be O(1) operation if cursor position has not changed from last search.

Thanks for taking care of the CL. I could not get to it quickly :(.